### DISCUSSION TITLE
W2ui grid Return custom error mesage isted of data from remote url

### DISCUSSION THREAD

**Discussion by kenyazilim** [vitmalina/w2ui#2369] [Q&A]
I'm using W2ui grid version 2 and trying to return an custom error message instead of record data.

The JSON I'm sending back to grid is

{error:true, message:"Test"}

but grid is not showing my message instead it is showing "HTTP error. See console for more details." in message poup. How can I change this message?

**Comment by vitmalina** [comment]
It works for me. Here is screenshot
<img width="626" alt="Screenshot 2023-04-24 at 10 14 36 AM" src="https://user-images.githubusercontent.com/561995/234069355-e96e971e-09b3-4605-9c41-b865e191a42b.png">

The error you are seeing suggests to me that your path might be wrong
